pdc/apps/messaging/apps.py

```diff
@@ -0,0 +1,32 @@
+#
+# Copyright (c) 2018 Red Hat
+# Licensed under The MIT License (MIT)
+# http://opensource.org/licenses/MIT
+#
+from django.apps import AppConfig
+from django.conf import settings
+from django.utils.module_loading import import_string
+
+
+class MessagingConfig(AppConfig):
+    name = 'pdc.apps.messaging'
+
+    def ready(self):
+        backend = settings.MESSAGE_BUS.get('BACKEND')
+        if backend:
+            cls = import_string(backend)
+            self.messenger = cls()
+        else:
+            # Drop this block in next version and instead directly fall on
+            # DummyMessenger.
+            from backends import capture, dummy, fedmsg, rhmsg
```

Comment:
Oh, looks like `django-admin makemigrations --check` calls `ready()` for apps.

It's perhaps better to make `MESSENGERS` map from the known strings to dotted path and use same code as above.